gmso/tests/test_itp.py

```diff
@@ -0,0 +1,10 @@
+from gmso.formats.itp import read_itp
+from gmso.tests.base_test import BaseTest
+from gmso.tests.utils import get_path
+
+
+class Testitp(BaseTest):
+    def test_itp(self):
```

Comment:
Keep up the tests here, we'll need a lot more to make sure this thing is reading all the values correctly.

Reply:
Missing tests for "PNB.itp"